database/migration/src/main/resources/changesets/postgresql_engine_schema.xml

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog objectQuotingStrategy="QUOTE_ALL_OBJECTS"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="kshakir (generated)" id="postgresql-engine-init-1" dbms="postgresql">
+        <createTable tableName="CALL_CACHING_AGGREGATION_ENTRY">
+            <column autoIncrement="true" name="CALL_CACHING_AGGREGATION_ENTRY_ID" type="SERIAL">
```

Comment:
FYI, I was worried that `autoIncrement="true"` would conflict with `type="SERIAL"`, since the separate `addAutoIncrement` instructions in the old migrations had this problem, but it appears this particular combination is just fine.